pkgs/top-level/all-packages.nix

```diff
@@ -10186,9 +10185,8 @@ with pkgs;
 
   opencv = opencv4;
 
-  openexr = openexr_2;
-  openexr_2 = callPackage ../development/libraries/openexr { };
```

Comment:
There should be an entry in `top-level/aliases.nix` that throws an error upon accessing `openexr_2` about the deprecation.

Reply:
And for ilmbase as well, as it is part of the same source tree and will get removed as well, right?

Reply:
If it was exposed in top-level then yes, otherwise no.

Reply:
As you choose to preserve `openexr_2` with an insecure mark, the above suggestion about placing `openexr_2` inside `aliases.nix` is no longer valid.